src/MissionEditor/MissionItemStatus.qml

```diff
@@ -74,53 +74,43 @@ Rectangle {
             QGCLabel { text: _azimuthText }
         }
 
-        QGCFlickable {
+        ListView {
+            id:    statusListView
+            model: missionItems
```

Comment:
Not a pain, if it isn't consistent it will become chaos. I'll open a new PR with fewer typos in the commit message too.